src/header/header.css

```diff
@@ -0,0 +1,101 @@
+header{
+    margin-top: 5rem;
+    height: calc(100vh - 5rem);
+    display: grid;
+    place-items: center;
+    background: var(--color-white);
+    transition: var(--transition);
+}
+
+.header-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+}
+
+.header-pf img{
+    width: 22rem;
+    height: 22rem;
+    object-fit: cover;
+}
+.header-pf{
+    border-radius: 50%;
+    overflow: hidden;
+    margin-bottom: 1.5rem;
+}
+
+
+.header-container p{
+    width: 35rem;
+    text-align: center;
+    margin: 0.6rem 0 2rem;
+}
+
+.header-cta{
+    display: flex;
+    align-items: center;
+    gap: 1.2rem;
+}
+
+.header-socials{
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    display: flex;
+    flex-direction: column;
+    gap:1rem;
+}
+
+.header-socials a{
+    width: 1.8rem;
+    aspect-ratio: 1/1;
+    background: var(--color-light);
+    color: var(--color-black);
+    display: grid;
+    place-items: center;
+    border-radius: var(--border-radius-3);
+    
+}
+
+.header-socials a:hover{
+    background: var(--color-primary);
+    color: var(--color-white);
+    transform: translateX(-0.5rem);
+}
+
+@media screen and (max-width : 1024px) {
+    header{
+        margin-top: 0;
+        height: 100vh;
+    }
+    
+}
+
+@media screen and (max-width : 600px) {
```

Comment:
in less than 400 is not responsive, could check it?!

Reply:
Here is why 

```
header-pf img {
    width: 100%;
    height: 100%;
    object-fit: cover;
}
```

here since height and width are 100% there is a moment when this page will get super small and so the image will expand vertically more than expected 

A possible solution to this:
You can set the height and width of the image on a smaller screen size to a fixed value like `100px`